google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -148,7 +148,6 @@ protected void showWarning(final String message) {
       public void run() {
         warningMessage.setText(message);
         warningMessage.setVisible(true);
-        warningMessage.updateUI();
```

Comment:
Why did you need to delete this call (and with cloudSdkPanel.updateUI() below)? I remember adding it to guarantee that the UI was properly redrawn when the message changed (specifically, to make sure that the window was automatically resized to fit the changed text).